cernopendata/modules/fixtures/data/records/cms-tools-Run1-datascience.json

```diff
@@ -47,7 +47,17 @@
     ],
     "relations": [
       {
-        "description": "The ouput dataset of this software tool is available in",
+        "description": "The tracker-hit-enriched AODSIM files produced by step 2 of this software tool are available in",
+        "recid": [
```

Comment:
We need to break this into separate elements, I'll take care of it while merging